differential/diffAtlas_DE.pl

```diff
@@ -218,7 +215,7 @@ sub run_microarray_differential_expression {
 
 	$logger->info( "Running differential expression analysis in R..." );
 	
-    my $tempDir = File::Spec->catdir( $ENV{ "HOME" }, "tmp" );
+    my $tempDir = File::Spec->catdir( $ENV{ "TMPDIR" }, "tmp" );
```

Comment:
Since this is working from reprocess and recalculations I'm assuming that the upstream process calling this will input the correct env var.